tests/foreman/cli/test_remoteexecution.py

```diff
@@ -168,7 +173,25 @@ def test_positive_view_dump(self):
 class RemoteExecutionTestCase(CLITestCase):
     """Implements job execution tests in CLI."""
 
-    @stubbed
+    @classmethod
+    def setUpClass(cls):
+        """Create an organization which can be re-used in tests."""
+        super(RemoteExecutionTestCase, cls).setUpClass()
+        cls.organization = make_org()
+        ssh.command(
+            '''echo 'echo "test"' > {0}'''.format(TEMPLATE_FILE)
+        )
+        cls.client = VirtualMachine(distro='rhel71')
```

Comment:
Do not see any removal procedure for that VM. Please, refer to our vm class and context manager:
`with VirtualMachine(distro='rhel71') as client:`
In case you perform that action in `setUpClass` like you do here, please do corresponding destroy action in `tearDownClass`